charger/pulsatrix.go

```diff
@@ -0,0 +1,289 @@
+package charger
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/evcc-io/evcc/api"
+	"github.com/evcc-io/evcc/util"
+	"github.com/gorilla/websocket"
```

Comment:
We're using

    github.com/nhooyr/websocket

Reply:
i was using gorilla since both WS-libs are in the go.mod file, but will change